Wikipedia/Categories/UITableViewCell+WMFLayout.m

```diff
@@ -0,0 +1,15 @@
+//  Created by Monte Hurd on 11/23/15.
+//  Copyright (c) 2015 Wikimedia Foundation. Provided under MIT-style license; please copy and modify!
+
+#import "UITableViewCell+WMFLayout.h"
+#import "NSProcessInfo+WMFOperatingSystemVersionChecks.h"
+
+@implementation UITableViewCell (WMFLayout)
+
+- (void)wmf_layoutIfNeededIfOperatingSystemVersionLessThan9_0_0 {
+    if ([[NSProcessInfo processInfo] wmf_isOperatingSystemVersionLessThan9_0_0]) {
+        [self layoutIfNeeded];
```

Comment:
Hmm I wasn't sure about downside - they've changed something (subtle?) about how constraints are taken into account between 8 and 9, I think, given that these bugs don't happen in 9. I thought scoping the layoutIfNeeded to 8 would be super-safe... I'm fine with removing the version check if you guys are. One nice thing about leaving it in, though - we would be we'd be notified that it's ok to remove these when we drop 8... wmf_isOperatingSystemVersionLessThan9_0_0 has a `WMF_DEPRECATED_WHEN_DEPLOY_AT_LEAST_9` check.